R/demo-crowding.R

```diff
@@ -1,23 +1,29 @@
 #' Access to Crowding Data
 #' 
-#' This function returns a data frame from Crowding in State Cancer Profiles
+#' This function returns a data frame about crowding demographics from State 
+#' Cancer Profiles.
```

Comment:
In general: keep every line under 80 characters. The function's description should use complete sentences. Use this description sentence formula wherever it's appropriate.